app/config/config.yml

```diff
@@ -29,6 +29,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    paths:
+        "%kernel.root_dir%/../custom/email_templates": custom_email_templates
```

Comment:
I don't like hardcoding the template path like this.  If you base it off of the % file_system_storage_path% parameter then it can be mounted anywhere on the server and not get deleted during upgrades.